crates/turbopack/src/module_options/module_rule.rs

```diff
@@ -74,8 +73,12 @@ pub enum ModuleType {
         transforms: EcmascriptInputTransformsVc,
         options: MdxTransformOptionsVc,
     },
-    Css(CssInputTransformsVc),
-    CssModule(CssInputTransformsVc),
+    CssGlobal,
+    CssModule,
+    Css {
```

Comment:
Should we rename to `CssInner`

Reply:
I think `CssGlobal` and `CssModule` are explicit enough that `Css` on its own is OK.